src/app/components/OptimoPromos/Content/index.jsx

```diff
@@ -0,0 +1,77 @@
+import React, { useContext } from 'react';
+import { string, bool } from 'prop-types';
+import { ServiceContext } from '#app/contexts/ServiceContext';
+import VisuallyHiddenText from '#app/legacy/psammead-visually-hidden-text/src';
+import moment from 'moment';
+import formatDuration from '#app/lib/utilities/formatDuration';
+
+const Content = ({
+  ariaLabelledBy,
+  mediaType,
+  mediaDuration,
+  headline,
+  isPhotoGallery,
+}) => {
+  const {
+    translations: { media: mediaTranslations },
+  } = useContext(ServiceContext);
+
+  if (!isPhotoGallery && !mediaType) {
+    // This span is a temporary fix for the a11y nested span's bug experienced in TalkBack, refer to the following issue: https://github.com/bbc/simorgh/issues/9652
+    return <span id={ariaLabelledBy}>{headline}</span>;
+  }
+
+  const getAnnouncedType = () => {
```

Comment:
this doesn't need to be a function

Reply:
or if it does, it should be defined outside of the component and anything it needs should be passed in as arguments so that the function is pure